rocAL/rocAL_pybind/amd/rocal/pipeline.py

```diff
@@ -173,6 +173,10 @@ def define_graph(self):
     def get_handle(self):
         return self._handle
 
+    def copyToExternalTensor(self, array,  multiplier, offset, reverse_channels, tensor_format, tensor_dtype):
```

Comment:
Are we introducing copyToExternalTensor in this mask PR?

Please align the arguments

Reply:
copyToExternalTensor C++ implementation is already introduced but we didn't need the pybind since copy_data was enough for classification trainings. Mask GPU trainings were lagging in perf so Shobana asked to introduce copyTensor to remove the perf difference